lib/tasks/sync_github.rake

```diff
@@ -0,0 +1,28 @@
+namespace :sync_github do
+
+  # This task fetch issues details and create author and book on our DB 
+  desc "Sync db from github"
+  task :author_details_from_github => :environment do
+    issues = Github.new.issues.list user: ENV['USERNAME'], repo: ENV['REPOSITORY']
```

Comment:
This line seems like it will only work if the repository is public, is there a reason you didn't use the `oauth` token to make authenticated requests, since you'll have it anyway?

Reply:
Yes, i used `oauth`  for private repository as well